patternbuilder.admin.inc

```diff
@@ -225,10 +266,112 @@ function patternbuilder_admin_bundle_form($form, &$form_state, $bundle = NULL) {
     $form['pattern_info'] += $info_items;
   }
 
-  // @todo: Add settings that can be changed / overridden in Drupal.
-  // Pattern status - active, private, deprecated, inactive.
-  // This would need to track status is forced / overridden so that a re-import
-  // did not change it.
+  // Pattern status.
+  $form['status_wrapper'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Pattern Status'),
+    '#collapsible' => TRUE,
+    '#collapsed' => FALSE,
+    '#description' => t('Configure the pattern status.'),
+  );
+
+  $ui_access = array(
```

Comment:
The initial thinking was to provide an easy way of adding other elements that would be controlled by a settings variable. Example: Pattern type.  The goal of these would to allow overriding until next import and to support a non-JSON schema driven value of these properties.

Currently only status is controlled.